views/admin/tpl/transaction_list.tpl

```diff
@@ -160,7 +160,11 @@ window.onload = function ()
                         <a href="Javascript:top.oxid.admin.editThis('[{$listitem->wdoxidee_ordertransactions__oxid->value}]');">[{$payment->oxpayments__oxdesc->value}]</a>
                     </td>
                     <td class="[{$listclass}]">
-                        <a href="Javascript:top.oxid.admin.editThis('[{$listitem->wdoxidee_ordertransactions__oxid->value}]');">[{$listitem->wdoxidee_ordertransactions__state->value}]</a>
+                        <a href="Javascript:top.oxid.admin.editThis('[{$listitem->wdoxidee_ordertransactions__oxid->value}]');">[{$listitem->wdoxidee_ordertransactions__state->value}]
+                          [{if $listitem->isPoiPiaPaymentMethod()}]
+                            ([{if !$listitem->wdoxidee_ordertransactions__parenttransactionid->value}]un[{/if}]matched)
```

Comment:
This is a bit hard to read, even though it works. I would just use a regular if/else with matched/unmached instead.

Reply:
👍